frontend/web/components/modals/CreateSegmentUsersTabContent.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import EnvironmentSelect from 'components/EnvironmentSelect'
+import IdentitySegmentsProvider from 'common/providers/IdentitySegmentsProvider'
+import PanelSearch from 'components/PanelSearch'
+import InfoMessage from 'components/InfoMessage'
+import InputGroup from 'components/base/forms/InputGroup'
+import Utils from 'common/utils/utils'
+import { Segment } from 'common/types/responses'
+import Icon from 'components/Icon'
+
+interface CreateSegmentUsersTabContentProps {
+  projectId: string | number
+  environmentId: string
+  setEnvironmentId: (environmentId: string) => void
+  identitiesLoading: boolean
+  identities: any
+  page: any
+  setPage: (page: any) => void
```

Comment:
The way the CreatesSegment is paging seems very strange rather than it just being a number. Probably one to fix as a future PR